cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -338,6 +327,23 @@
             display: none !important;
         }
     } );
+
+    // Hide fallback by default.
+    &_fallback {
+        display: none;
+        padding: 8px;
+        padding-right: 0;
+
+        // Only show fallback if JavaScript (JS) is not enabled.
+        .no-js & {
+            display: block;
+        }
+    }
+}
+
+// Only show global search if JavaScript (JS) is enabled.
+html.js .m-global-search_trigger {
```

Comment:
Yea, if I recall this is because we have the js/no-js class in the header component itself (see [here](https://github.com/cfpb/consumerfinance.gov/blob/0fa6c94ebd6c9c28a8539d620dd76d2cf286deae/cfgov/jinja2/v1/_includes/organisms/header.html#L73)), which is the parent of the search.

Reply:
Huh. Do you remember why we do that? Regardless, would it matter? Does that class on the header not stay in sync with whatever is on `html`?

Reply:
I think it's so you can change the class on `html` instead of digging around for the class on the header, and then swap it to view the two states without needing to switch js on/off and reload the page. Jimmy had added it [here](https://github.com/cfpb/consumerfinance.gov/commit/0a57d03c9d409e7ac93f2ce7929b5f8f3460b6cb). We should dig into whether it's still applicable.